frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/report/simulated-region-overview-behavior-report.component.html

```diff
@@ -0,0 +1,141 @@
+<h5>Berichte</h5>
+
+<div class="rounded-top" [class.bg-primary]="!createReportCollapsed">
+    <button
+        type="button"
+        class="btn btn-primary btn-sm w-100"
+        (click)="createReportCollapsed = !createReportCollapsed"
+    >
+        Bericht erstellen
+    </button>
+</div>
+
+<div
+    #collapse="ngbCollapse"
+    [(ngbCollapse)]="createReportCollapsed"
+    class="border border-primary rounded-bottom p-2"
+>
+    <select class="form-select no-validation" [(ngModel)]="selectedInformation">
+        <option value="noSelect" disabled hidden selected>
+            Wählen Sie die Information des Berichts
+        </option>
+        <option
+            *ngFor="let information of reportableInformations"
+            [value]="information"
+        >
+            {{ reportableInformationTranslationMap[information] }}
+        </option>
+    </select>
+
+    <div class="row align-items-center justify-content-between my-2">
+        <div class="col-6">
+            <div class="input-group input-group-sm">
+                <label
+                    class="btn"
+                    [class]="
+                        once.checked ? 'btn-secondary' : 'btn-outline-secondary'
+                    "
```

Comment:
I did not manage to find out the cause for this, but the button colors are not as I would expect them to be:

Clicking this ("Einmalig") or the other ("Regelmäßig") button does not cause the colors to change. However, selecting another report type in the `<select>` element *sometimes* does change which button is highlighted, but I can't change the choice afterwards, too.

Reply:
Are you still able to reproduce this? It seems to work fine for me now...

Reply:
Yes, it still does not work (on commit 76c5a2bc4338137206eeddc56db29c91609e1f5b, using Firefox Developer Edition 112 as well as Chromium 111)

Reply:
Ok, I can reproduce it on 76c5a2bc4338137206eeddc56db29c91609e1f5b, but on the current head it seems fine.

Reply:
I'm gonna check it. As far as I can tell, it only seems to be a visual bug on https://github.com/hpi-sam/digital-fuesim-manv/commit/76c5a2bc4338137206eeddc56db29c91609e1f5b, the underlying input elements are (un-)checked as expected

Reply:
No, it does not work on 3f6744a739e1af5ef8a94bbf8333c782e9507fd0, too. I've tested in Firefox DE and Chromium as before.

However, the highlighted button does change when I trigger some other changes in the UI (like selecting the information type in the `<select>`, toggling the switches in the radiogram sidebar, ...), so a possible fix could be to use an explicit backing field with `[(ngModel)]` for the value of the radio buttons instead of the component ref.

Reply:
Fixed in 5e55ee610c03aa7e62a3086487b6512829ae5718

Reply:
With https://github.com/hpi-sam/digital-fuesim-manv/pull/780/commits/5e55ee610c03aa7e62a3086487b6512829ae5718 it seems to be working in both the small popup and the large modal.